x/feesplit/keeper/integration_test.go

```diff
@@ -933,7 +936,7 @@ func checkEthTx(priv *ethsecp256k1.PrivKey, msgEthereumTx *evmtypes.MsgEthereumT
 	return res
 }
 
-func prepareCosmosTx(priv *ethsecp256k1.PrivKey, gasPrice *sdk.Int, msgs ...sdk.Msg) []byte {
+func prepareCosmosTx(priv *ethsecp256k1.PrivKey, gasPrice *sdkmath.Int, msgs ...sdk.Msg) []byte {
```

Comment:
would be consistent with the import. On other files `math` is used and here `sdkmath`

Reply:
it is conflicted with an already imported math package in that file